build.gradle

```diff
@@ -89,6 +89,11 @@ dependencies {
     compile group: 'com.google.api.grpc', name: 'proto-google-common-protos', version: '2.10.0'
     compile group: 'com.google.protobuf', name: 'protobuf-java-util', version: '3.21.9'
     compile group: 'com.google.oauth-client', name: 'google-oauth-client', version: '1.35.0'
+    compile group: 'org.projectlombok', name: 'lombok', version: '1.18.30'
```

Comment:
We should only need the compileOnly dependency. We don't want to include this at runtime.